feature-libs/customer-ticketing/root/events/customer-ticketing.events.ts

```diff
@@ -0,0 +1,15 @@
+import { CxEvent } from '@spartacus/core';
+
+export class GetTicketQueryResetEvent extends CxEvent {
+  /**
+   * Event's type
+   */
```

Comment:
Are these comments needed?

Reply:
not really, but it was there for all others events. should I remove it?

Reply:
There are not adding any value.  Please remove

Reply:
got it